clar2wasm/src/standard/standard.wat

```diff
@@ -1413,6 +1421,7 @@
         (local.set $length)
 
         (local.set $i (i32.const 0))
+
```

Comment:
Why did you add those weird newlines here and there?

Reply:
there were logging statements, accidently left a lot of lines